airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -488,6 +510,11 @@ protected static ConnectionSearch toConnectionSearch(final WebBackendConnectionS
         .status(webBackendConnectionSearch.getStatus());
   }
 
+  @VisibleForTesting
+  protected static List<StreamDescriptor> getStreamsToReset(final CatalogDiff catalogDiff) {
```

Comment:
```suggestion
  static List<StreamDescriptor> getStreamsToReset(final CatalogDiff catalogDiff) {
```
This doesn't need `protected`, since the test is in the same package as this class. So we can just do package-private (i.e. no keyword), which limits this to only being used by classes in the same package